documentation/experimental_features.md

```diff
@@ -6,7 +6,135 @@ while iterating on new functionality. Almost all experimental features are
+patterns for easily listing multiple policies. For example, `myproject::*`
+includes both the `myproject::admin` and `myproject::sshkeys` policies.
+
+The following configuration file makes the `myproject::admin` and
+`myproject::sshkeys` policies available to the project:
+```
+---
+# bolt-project.yaml
+name: myproject
+policies:
+- myproject::admin
+- myproject::sshkeys
+```
+
+### Creating new policies 
+
+You can use Bolt to create a new policy in your project and add it to the
+`policies` key in your project configuration. For example, to create a new
+project-level policy named `myproject::user`, run:
+
+* _*nix shell command_
+  ```
+  bolt policy new myproject::user
+  ```
+
+* _PowerShell cmdlet_
+  ```
+  New-BoltPolicy -Name myproject::user
+  ```
+
+Bolt creates an empty class in the project's `manifests/` directory that you can
+populate with code. Policy names must follow [class naming
+conventions](https://puppet.com/docs/puppet/7/lang_reserved.html#classes-and-defined-resource-type-names).
+
+
+Example output:
+```
+Created policy 'myproject::user' at '/Users/puppet.user/myproject/manifests/user.pp'
+```
+
+Create policies manually by:
+
+1. Adding the file to a module's or project's `manifests/` directory.
+2. Modifying the project's `bolt-project.yaml` to include the policy in the
+   `policies` key.
+
+### Listing available policies
+
+You can list available policies for the project:
+
+* _*nix shell command_ 
+  ```
+  bolt policy show 
+  ```
+
+* _PowerShell cmdlet_ 
+  ``` 
+  Get-BoltPolicy 
+  ```
+
+Example output:
+```
+Policies
+  myproject::admin
+  myproject::sshkeys
+
+Modulepath
+  /Users/puppet.user/myproject/modules:/Users/puppet.user/myproject/.modules
+```
+
+### Applying policies to targets
+
+Applying policies is similar to [applying Puppet
+code](https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html), with
+the addition that you can apply one or more policies at a time. The Bolt
+commands for applying policies accept a single policy name or a comma-separated
+list of policy names to apply to a list of one or more targets. 
+
+For example, to apply both `myproject::admin` and `myproject::sshkeys` policies
+to a target:
+
+* _*nix shell command_
+  ```
+  bolt policy apply myproject::admin,myproject::sshkeys -t mytarget                                
+  ```
+
+* _PowerShell cmdlet_
+  ```
+  Invoke-BoltPolicy -Name myproject::admin,myproject::sshkeys -Targets mytarget
+  ```
+
+Example output:
+```
+Starting: install puppet and gather facts on mytarget
+Finished: install puppet and gather facts with 0 failures in 6.51 sec
+Starting: apply catalog on mytarget
+Started on mytarget...
+Finished on mytarget:
+  changed: 1, failed: 0, unchanged: 0 skipped: 0, noop: 0
+Finished: apply catalog with 0 failures in 8.59 sec
+Successful on 1 target: mytarget
+Ran on 1 target in 15.2 sec
+```
+
+Before applying policies, Bolt installs the `puppet-agent` package and collects
```

Comment:
Would it be work linking to documentation about the nuances of apply_prep here (just blanket saying it installs the agent seems somewhat inaccurate and scary) Maybe if we dont want to find a link just say something like `Bolt ensures the puppet-agent packages is available on the target and collects facts`?

Reply:
How does this sound?
> Before applying policies, Bolt installs uses the `apply_prep` function to collect 
  facts and ensure that the `puppet-agent` package is available on each target.
  For more information on how Bolt uses `apply_prep`, see
  [Applying manifest blocks]applying_manifest_blocks.md#applying-manifest-blocks-from-a-puppet-plan

Reply:
That sounds good to me, does this check out with you @beechtom? I believe we can go ahead and merge this after this change unless @lucywyman wants to add anything.

Reply:
@rileynewton Looks good to me 👍 Before merging you'll need to squash all your commits down to a single commit.